### PR TITLE
menu section

### PR DESCRIPTION
**Pull request by ArtemG22**
http://lambdarest.surge.sh/

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -351,4 +355,95 @@ html {
...
+/* MENU SECTION */
+/******************************/
+
+.menu-section {
+  background: url('../gallery/banners/menu-banner.png');
+  background-repeat: repeat;
+  padding: 10rem;
+  width: 100%;
+  box-shadow: inset 0 0 6rem rgb(0, 0, 0, 0.1);
+  gap: 16.8rem;
```
Comment (ynworld) [inline css/style.css:370]: Don't think this does anything.

Comment (ynworld) [inline css/style.css]: Don't think this does anything.

File: index.html [hunk 2]
```diff
@@ -126,6 +127,121 @@ <h3 class="heading-secondary heading-light">Fine ingredients</h3>
...
+          <h4 class="heading-secondary heading-dark">Appetisers</h4>
+          <img src="gallery/decoration/decoration-2.png" class="decoration-menu" alt="decorative element">
+          <div class="menu-items">
+            <p>Tzatsiki</p>
+            <p>$3.99</p>
+          </div>
+          <div class="line-decoration"></div>
+          <blockquote class="description-of-menu-items menu-indent">Refreshing traditional cucumber and garlic yoghurt
+            dip.
+          </blockquote>
```
Comment (ynworld) [inline index.html]: Seems like an `article` element.

File: css/style.css [hunk 3]
```diff
@@ -351,4 +355,95 @@ html {
...
+.line-decoration {
+  background-color: #cc9900;
+  height: 0.1rem;
+  opacity: 30%;
+  margin-bottom: 1.2rem;
+}
+
+.menu-indent {
+  margin-bottom: 3.6rem;
+}
```
Comment (ynworld) [inline css/style.css]: Why not just add this margin to `description of menu items`
image

File: index.html [hunk 4]
```diff
@@ -130,114 +130,146 @@ <h3 class="heading-secondary heading-light">Fine ingredients</h3>
...
+            <article class="dish and description">
+              <article class="menu-items">
+                <p>Tzatsiki</p>
+                <p class="price">$3.99</p>
+              </article>
+              <div class="line-decoration"></div>
+              <article class="description-of-menu-items">Refreshing traditional cucumber and garlic yoghurt
+                dip.
+              </article>
+            </article>
```
Comment (ynworld) [inline index.html]: This is all **one** article with a title, price and description.

File: index.html [hunk 5]
```diff
@@ -130,114 +130,146 @@ <h3 class="heading-secondary heading-light">Fine ingredients</h3>
...
+            <article class="dish and description">
```
Comment (ynworld) [inline index.html]: Classes should refer to what you are using in the style sheet. They shouldn't be used as element descriptors. Consider simplifying the nested elements. If it has no styles and isn't helping with the layout, then it's probably not needed.

File: index.html [hunk 6]
```diff
@@ -126,6 +127,153 @@ <h3 class="heading-secondary heading-light">Fine ingredients</h3>
       </div>
     </section>
 
+    <section class="menu-section">
+      <div class="menu-columns">
+        <div>
+          <section>
+            <h4 class="heading-secondary heading-dark">Appetisers</h4>
+            <img src="gallery/decoration/decoration-2.png" class="decoration-menu" alt="decorative element">
```
Comment (magicsd) [inline index.html]: Usually we start sentence with a capital letter
```suggestion
            <img src="gallery/decoration/decoration-2.png" class="decoration-menu" alt="Decorative element">
```

File: index.html [hunk 7]
```diff
@@ -126,6 +127,153 @@ <h3 class="heading-secondary heading-light">Fine ingredients</h3>
...
+            <article>
+              <div class="menu-items">
+                <p>Tzatsiki</p>
+                <p>3.99</p>
+              </div>
+              <div class="line-decoration"></div>
+              <p class="description-of-menu-items">Refreshing traditional cucumber and garlic yoghurt
```
Comment (magicsd) [inline index.html]: Checkout the pattern logic:
```css
.block {
  .block-heading {}

  .block-items {
    .block-items-description {}

    .block-items-image {}
  }
}

```
```suggestion
              <p class="menu-items-description">Refreshing traditional cucumber and garlic yoghurt
```

`.description-of-menu-items` looks not reusable and not composable

File: index.html [hunk 8]
```diff
@@ -126,6 +127,153 @@ <h3 class="heading-secondary heading-light">Fine ingredients</h3>
...
+                dip.
+              </p>
+            </article>
+            <article>
+              <div class="menu-items">
+                <p>Aubergine Salad</p>
+                <p>$5.50</p>
+              </div>
```
Comment (magicsd) [inline index.html]: Looks like [description list](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl)

File: css/style.css [hunk 9]
```diff
@@ -184,6 +184,10 @@ html {
   color: #fff;
 }
 
+.indent {
+  padding-top: 2.8rem;
+}
```
Comment (magicsd) [inline css/style.css]: Looks redundant. Why do you need it?

File: css/style.css [hunk 10]
```diff
@@ -351,4 +355,92 @@ html {
...
+  top: 0;
+  left: 0;
+  position: absolute;
+  background: linear-gradient(to bottom right, rgba(85, 85, 85, 0.2) -100%, transparent 70%);
+  z-index: -1;
+}
+
+.menu-columns {
+  display: grid;
+  grid-template-columns: 43.2rem 43.2rem;
```
Comment (magicsd) [inline css/style.css]: Maybe it'll be useful to make these columns just of the same width and be defined automatically?

File: css/style.css [hunk 11]
```diff
@@ -351,4 +355,92 @@ html {
...
+}
+
+.menu-items-active {
+  border: 0.05rem solid #cc9900;
+  position: absolute;
+  width: 45.4rem;
+  height: 8rem;
+  opacity: 50%;
+  margin-left: -1rem;
+  margin-top: -1rem;
```
Comment (magicsd) [inline css/style.css]: If you have to use negative margin, then probably you're doing something wrong

File: css/style.css [hunk 12]
```diff
@@ -351,4 +355,92 @@ html {
...
+  font-family: "Yeseva One", sans-serif;
+  font-size: 2rem;
+  font-weight: 400;
+  color: #cc9900;
+  justify-content: space-between;
+  margin-bottom: 0.8rem;
+}
+
+.menu-items-active {
+  border: 0.05rem solid #cc9900;
```
Comment (magicsd) [inline css/style.css]: Why not in pixels?

File: index.html (hunk 7 above)
Comment (ArtemG22) [reply index.html]: There are still doubts about the implementation of the paragraph of the correct structure. I am not closing this comment.

File: css/style.css [hunk 13]
```diff
@@ -407,13 +411,12 @@ html {
 }
 
 .menu-items-active {
-  border: 0.05rem solid #cc9900;
+  border: 0.5px solid #cc9900;
```
Comment (magicsd) [inline css/style.css]: I've never seen border width less than 1px 😅

File: css/style.css [hunk 14]
```diff
@@ -362,6 +362,8 @@ html {
   background-repeat: repeat;
   padding-top: 10rem;
   padding-bottom: 6.4rem;
+  padding-left: 28rem;
+  padding-right: 28rem;
```
Comment (ynworld) [inline css/style.css]: Why not use the property shorthand?
```suggestion
padding: 10rem 28rem 6.4rem;
```

Comment (ArtemG22) [reply css/style.css]: fixed

**Review by magicsd — APPROVED** [review]

**Pull request merged**